HARK/ConsumptionSaving/ConsIndShockModel.py

```diff
@@ -53,6 +56,8 @@
     CRRAutility_invP,
     CRRAutility_inv,
     CRRAutilityP_invP,
+    jump_to_grid,
+    jump_to_grid_fast,
```

Comment:
More descriptive names would be useful for extensions/alternative uses. In the past, we've used `_fast` to refer to numba-fied methods that do the same as their reference methods, but here, both are using `numba.jit`.

Reply:
Ok got it.